jsk_maps/launch/start_map_eng2.launch

```diff
@@ -24,6 +25,14 @@
     <remap from="initialpose_out" to="initialpose" />
   </node>
 
+  <!-- change floor server -->
+  <node name="change_floor_server" pkg="jsk_maps" type="change-floor-server.l">
+    <rosparam subst_value="true">
+      base_frame_id: $(arg base_frame_id)
+      move_base_node: move_base
```

Comment:
can you also set `$(arg move_base_node)` in this launch?

Reply:
PR2's move base node is not move_base, move_base_node.
https://github.com/jsk-ros-pkg/jsk_demos/blob/master/jsk_maps/src/change-floor-server.l#L15